src/plugins/tabs/_screen-sm-max.scss

```diff
@@ -9,20 +9,29 @@
 %tabs-screen-sm-max-details {
 	> {
 		details {
-			border-width: $tablist-screen-sm-details-border-width;
-      border-style: $tablist-screen-sm-details-border-style;
-      border-color: $tablist-screen-sm-details-border-color;
-			border-top: 0;
```

Comment:
I collapsed this into `border-width` which became `$tablist-screen-sm-details-border-width: 0 1px 1px 1px;`